examples/static_react_task/README.md

```diff
@@ -21,7 +21,7 @@ This task is configured using [Hydra](https://hydra.cc/) - details about using h
 In this script, we set certain configuration variables by default in every run:
 ```python
 defaults = [
-    {"mephisto/blueprint": BLUEPRINT_TYPE},
+    {"mephisto/blueprint": BLUEPRINT_TYPE_<>},
```

Comment:
typo? or intentional `<>`

Reply:
Ah intentional but I didn't realize the context - had way too much flying around on this change. Thanks for the catch, I'd prefer `STATIC_REACT` over `<>` in that case as well.